Dockerfile

```diff
@@ -12,6 +12,11 @@ RUN rm -rf /srv/ttn-lorawan/lorawan-frequency-plans/.git
 COPY data/lorawan-webhook-templates /srv/ttn-lorawan/lorawan-webhook-templates
 RUN rm -rf /srv/ttn-lorawan/lorawan-webhook-templates/.git
 
+COPY data/lorawan-devices /tmp/lorawan-devices
+RUN rm -rf /tmp/lorawan-devices/.git && \
+  mkdir -p /srv/ttn-lorawan/device-repository && \
```

Comment:
This dirname may be a bit confusing, maybe we should make clear that it contains the index, not the entire repository.